caddyhttp/proxy/upstream.go

```diff
@@ -436,6 +436,7 @@ func parseBlock(c *caddyfile.Dispenser, u *staticUpstream, hasSrv bool) error {
 		u.upstreamHeaders.Add("Host", "{host}")
 		u.upstreamHeaders.Add("X-Real-IP", "{remote}")
 		u.upstreamHeaders.Add("X-Forwarded-Proto", "{scheme}")
+		u.upstreamHeaders.Add("Forwarded", "for={remote}; host={host}; proto={scheme}")
```

Comment:
As per https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded if the {remote} is an IPV6 address it needs to be quoted and in square brackets [] .

Reply:
Oh sorry I missed that